docs/upgrade/xenial_upgrade_in_place.rst

```diff
@@ -34,16 +32,23 @@ Before performing the upgrade, please perform all the steps outlined in
   importance that your *Admin Workstation* and your servers use SecureDrop
   0.12.0. Older releases of SecureDrop do not not support Ubuntu 16.04.
 
-Upgrade and validate the *Monitor Server*
------------------------------------------
+                                                                                
+We expect that the upgrade should take under 8 hours to complete, but recommend 
+that you plan for 2 days of downtime for your instance, in case of error. You
+should coordinate the timing of the upgrade with the people responsible for 
+checking for submissions. If they have ongoing conversations with sources, or 
+are expecting submissions, it may be necessary to reschedule the upgrade. You 
+should also consider communicating the planned downtime and the reason for it 
+on your instance's landing page.
 
-On your *Admin Workstation*, open a terminal by selecting
-**Applications > Favorites > Terminal**. Then connect to the *Monitor Server*
-with the command ``ssh mon``.
 
-Before running the ``do-release-upgrade`` command, you must edit the
-``/etc/update-manager/release-upgrades``, changing `Prompt=never` to
-`Prompt=lts`. **[ED - this won't be necessary when #4104 lands]**
+Step 1: upgrade the *Monitor Server*
+------------------------------------
```

Comment:
Upper case after the ":" (i.e. "Step 1: Upgrade") looks a bit cleaner to me; that's the formatting I'm using in the other doc. I'd suggest using that here, unless you have a strong preference (no precedent in docs AFAICT).

Reply:
because it's after the colon I'm treating it as part of the same sentence, capitals seem forced. I have no authority to appeal to other than my own sense of the aesthetic